addOns/automation/src/main/javahelp/org/zaproxy/addon/automation/resources/help/contents/job-ascan.html

```diff
@@ -44,6 +44,10 @@ <H2>YAML</H2>
         threshold:                     # String: The Alert Threshold for this rule, one of Off, Low, Medium, High, default: Medium
 </pre>
 
+<p>
+The policy can be one defined by a previous <a href="job-ascanpolicy.html">activeScan-policy</a> job, or by a scan policy file
+that has been put in the expected directory.
```

Comment:
Is "expected directory" the one required by ZAP or one defined by the user (path)? Can we make this more clear/user friendly somehow?

Reply:
Yeah, was wonding about that.
Its the `policy` directory under the ZAP home dir, as per https://www.zaproxy.org/faq/what-is-the-default-directory-that-zap-uses/
Whats the best way to describe that?

Reply:
```suggestion
that has been put in the expected directory (<code>policy</code> within ZAP's HOME directory).
```

With "ZAP's HOME directory" linked...

Reply:
Updated